_inc/client/security/jetpack-backup.jsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { LoadingCard } from './loading-card';
+import Banner from 'components/banner';
+import SettingsCard from 'components/settings-card';
+
+export class JetpackBackup extends Component {
+	static propTypes = {
+		siteRawUrl: PropTypes.string,
+		rewindState: PropTypes.string,
+	};
+
+	static defaultProps = {
+		siteRawUrl: '',
+		rewindState: '',
+	};
+
+	getRewindMessage() {
+		const { siteRawUrl, rewindStatus } = this.props;
+		const rewindState = get( rewindStatus, [ 'state' ], false );
+
+		switch ( rewindState ) {
+			case 'provisioning':
+				return {
+					title: __( 'Provisioning' ),
+					icon: 'info',
+					description: __( 'Jetpack Backup is being configured for your site.' ),
+					url: '',
+				};
+			case 'awaiting_credentials':
+				return {
+					title: __( 'Awaiting credentials' ),
+					icon: 'notice',
+					description: __(
+						'You need to enter your server credentials to finish configuring Jetpack Backup.'
+					),
+					url: 'https://wordpress.com/settings/security/' + siteRawUrl,
+				};
+			case 'active':
+				return {
+					title: __( 'Active' ),
+					icon: 'checkmark-circle',
+					description: __( 'Your site is being backed up.' ),
+					url: 'https://wordpress.com/activity-log/' + siteRawUrl,
+				};
+			default:
+				return {
+					title: '',
+					icon: '',
+					description: '',
+					url: '',
+				};
+		}
+	}
+
+	getBanner = () => {
+		const { title, icon, description, url } = this.getRewindMessage();
```

Comment:
Should we display no banner if here is no rewind message?

Reply:
I made a change to add in handling for VaultPress, so the code will never get here now when there is no rewind message.